docarray/typing/url/filetypes.py

```diff
@@ -117,3 +117,67 @@
     'vcs',
     'vcf',
 ]
+
+IMAGE_FILE_FORMATS = [
+    '.icb',
+    '.psd',
+    '.im',
+    '.bmp',
+    '.ppm',
+    '.dcx',
+    '.gbr',
+    '.emf',
+    '.jpeg',
+    '.tiff',
+    '.rgb',
+    '.tif',
+    '.apng',
+    '.dds',
+    '.png',
+    '.ps',
+    '.ico',
+    '.fli',
+    '.iim',
+    '.ftu',
+    '.vst',
+    '.pnm',
+    '.h5',
+    '.j2k',
+    '.xpm',
+    '.ftc',
+    '.icns',
+    '.gif',
```

Comment:
Where does this list of extensions come from? Is it the extension supported by Pillow? Are we sure that something like `.gif` works for our purposes?

Reply:
Yes these are extensions supported by pillow. I had run this code to find a list of all the supported extensions
```
from PIL import Image

exts = Image.registered_extensions()
supported_extensions = {ex for ex, f in exts.items() if f in Image.OPEN}
```

The list consisted of formats like `mpeg` and `mpg` too, but I removed them since they were video formats. Looks like I forgot to remove the GIF format. Pushing a commit to remove gif. Thanks a lot :)

Reply:
perfect